contrib/python-pytest-asyncio/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-pytest-asyncio"
+pkgver = "0.23.4a2"
```

Comment:
0.23.3 is the latest stable version

Reply:
The problem is that CI forces to use the latest. How to skip it?

Reply:
ignore = ["*a*"] in update.py